boards/atsamv71_xult/memory.x

```diff
@@ -1,7 +1,16 @@
 MEMORY
 {
   FLASH : ORIGIN = 0x400000, LENGTH = 2M
-  RAM : ORIGIN = 0x20400000, LENGTH = 256K /* 256K or 384K; p. 54 */
+  /* Total RAM is 256K or 384K; p. 54 */
+  CAN : ORIGIN = 0x20400000, LENGTH = 64K
+  RAM : ORIGIN = 0x20410000, LENGTH = 192K
+}
+
+SECTIONS {
+  .can (NOLOAD) :
+  {
+    *(.can .can.*);
+  } > CAN
```

Comment:
Thanks for the double post, github.

Reply:
It's like a dedicated `.bss` section, any symbol from `.can*` input sections should be mapped to `.can` output section in the CAN memory area. NOLOAD provides the expectation that this memory should not be initialized (usage through `MaybeUninit` is sound and `SharedMemory` is using it underneath). In general this is needed because CAN RAM has to be in a predictible memory region that matches platform's configuration. Eg. SAMD/E5x hardcodes it to 0x2000_XXXX and x7x is configurable by `CCFG_{CAN0, SYSIO}`. MCAN itself is unaware of this position, it is using 16-bit addressing internally.